src/ray/object_manager/pull_manager.cc

```diff
@@ -30,9 +30,7 @@ uint64_t PullManager::Pull(const std::vector<rpc::ObjectReference> &object_ref_b
       // We don't have an active pull for this object yet. Ask the caller to
       // send us notifications about the object's location.
       objects_to_locate->push_back(ref);
-      it = object_pull_requests_
-               .emplace(obj_id, ObjectPullRequest(get_time_() + pull_timeout_ms_ / 1000))
-               .first;
+      it = object_pull_requests_.emplace(obj_id, ObjectPullRequest(get_time_())).first;
```

Comment:
Can you also add a comment to explain why the next pull time should be set to current time?
```suggestion
      it = object_pull_requests_.emplace(obj_id, ObjectPullRequest(/*next_pull_time=*/get_time_())).first;
```